roles/accprovision/scripts/process_string_to_yaml.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python3
+
+import json
+import yaml
+import copy
+import os
+
+def main():
+    cmap_input_path = os.getenv('CONFIGMAP_INPUT_FILE')
+    cmap = dict()
+    input_dict = dict()
+    with open(cmap_input_path, 'r') as stream:
+        cmap = yaml.safe_load(stream)
+        yaml_cmap_intermediate = cmap['data']['spec']
+        input_dict = yaml.safe_load(yaml_cmap_intermediate)
+        yaml_cmap = input_dict['acc_provision_input']
+    crd_input = os.getenv('CRDINPUT')
+    yaml_crd = yaml.safe_load(crd_input)
+
+    crd_definition = os.getenv('CRD_DEFINITION')
+    crd_definition = yaml.safe_load(crd_definition) 
+    deep_merge(yaml_crd, yaml_cmap, crd_definition)
+
+    # Write configmap file
+    input_dict['acc_provision_input'] = yaml_crd
+    cmap['data']['spec'] = json.dumps(input_dict)
+    cmap['metadata'].pop('managedFields', None)
+    cmap['metadata'].pop('annotations', None)
+    cmap['metadata'].pop('creationTimestamp', None)
+    cmap['metadata'].pop('resourceVersion', None)
+    cmap['metadata'].pop('uid', None)
+    with open(cmap_input_path, 'w') as outfile:
+        json.dump(cmap, outfile)
+
+    # Write acc_provision_input to file
+    acc_prov_dir = os.getenv('ACCPROVDIR')
+    acc_prov_file = os.getenv('ACCPROVFILE')
+    acc_prov_file_path = os.path.join(acc_prov_dir, acc_prov_file)
```

Comment:
can be clubbed above 2 vars

Reply:
We need the ACCPROVDIR variable in other places too, so made two separate variables

Reply:
i was referring to make it like acc_prov_file_path = os.path.join(os.getenv('ACCPROVDIR'), os.getenv('ACCPROVFILE'))

I don't see acc_prov_dir  and acc_prov_file variables used anywhere else

Reply:
It's a minor comment. Just trying to decrease the line of codes and number of variables. :)

Reply:
Ah I see what you mean, sure let me merge those. Thanks